virt/virt-4-11-release-notes.adoc

```diff
@@ -0,0 +1,84 @@
+
+Learn more about xref:../virt/about-virt.adoc#about-virt[what you can do with {VirtProductName}].
+
+include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
+
+[id="virt-guest-os"]
+=== Supported guest operating systems
+//CNV-16390 Supported guest operating systems
+To view the supported guest operating systems for {VirtProductName}, refer to link:https://access.redhat.com/articles/973163#ocpvirt[Certified Guest Operating Systems in Red Hat OpenStack Platform, Red Hat Virtualization and OpenShift Virtualization].
+
+[id="virt-4-11-inclusive-language"]
+== Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
+
+[id="virt-4-11-new"]
+== New and changed features
+
+//CNV-16067 CNV revalidated on a 3-node compact cluster
+
+[id="virt-4-11-quick-starts"]
+=== Quick starts
+
+* Quick start tours are available for several {VirtProductName} features. To view the tours, click the *Help* icon *?* in the menu bar on the header of the {VirtProductName} console and then select *Quick Starts*. You can filter the available tours by entering the `virtualization` keyword in the *Filter* field.
+
+[id="virt-4-11-installation-new"]
+=== Installation
+
+
+[id="virt-4-11-networking-new"]
+=== Networking
+
+
+[id="virt-4-11-storage-new"]
+=== Storage
+
+
+[id="virt-4-11-web-new"]
+=== Web console
+
+// NOTE: Comment out deprecated and removed features (and their IDs) if not used in a release
+[id="virt-4-11-deprecated-removed"]
+== Deprecated and removed features
+
+[id="virt-4-11-deprecated"]
+=== Deprecated features
+
+Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
+
+[id="virt-4-11-removed"]
+=== Removed features
+
+Removed features are not supported in the current release.
+
+// NOTE: no notable technical changes in 4.9, commenting out to retain
+//[id="virt-4-11-changes"]
+//== Notable technical changes
+
+[id="virt-4-11-technology-preview"]
+== Technology Preview features
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use. Note the following scope of support on the Red Hat Customer Portal for these features:
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+[id="virt-4-11-bug-fixes"]
+== Bug fixes
+
+[id="virt-4-11-known-issues"]
+== Known issues
```

Comment:
Did you clone the previous release notes file? We do not remove anything in the **Known issues** section because some issues might not be resolved for the next release and we need to keep them in the release notes. 

It might also be helpful to keep the **Deprecated features** and the **Technology Preview features** from the previous release and delete them after making sure they will not be supported or go GA in the next release respectively.